src/main/resources/sim_api.h

```diff
@@ -125,10 +127,17 @@ template <class T> class sim_api_t {
   }
   void init_channels() {
     pid_t pid = getpid();
+    char cwd[PATH_MAX];
+    if (getcwd(cwd, sizeof(cwd)) != NULL) {
+       printf("Current working dir: %s\n", cwd);
+    } else {
+       perror("getcwd() error");
+       return;
+    }
     std::ostringstream in_ch_name, out_ch_name, cmd_ch_name;
-    in_ch_name  << std::dec << std::setw(8) << std::setfill('0') << pid << ".in";
-    out_ch_name << std::dec << std::setw(8) << std::setfill('0') << pid << ".out";
-    cmd_ch_name << std::dec << std::setw(8) << std::setfill('0') << pid << ".cmd";
+    in_ch_name  << cwd << "/" << std::dec << std::setw(8) << std::setfill('0') << pid << ".in";
```

Comment:
Can you please say why you add the *cwd* to the names here.
I didn't notice but is something else changed to adjust for these new names

Reply:
I will add the following comment in code:
`// As VSIM relative path management is a mess, enforce the use of absolute paths for all channels`